Purchases/Misc/Obsoletions.swift

```diff
@@ -280,3 +280,15 @@ public extension Purchases {
 @available(watchOS, obsoleted: 1, renamed: "StoreTransaction")
 @available(macOS, obsoleted: 1, renamed: "StoreTransaction")
 @objc(RCTransaction) public class Transaction: NSObject { }
+
+public extension Package {
+    /**
+     `SKProduct` assigned to this package. https://developer.apple.com/documentation/storekit/skproduct
+    */
+    @available(iOS, obsoleted: 1, renamed: "storeProduct", message: "Use storeProduct instead")
+    @available(tvOS, obsoleted: 1, renamed: "storeProduct", message: "Use storeProduct instead")
+    @available(watchOS, obsoleted: 1, renamed: "storeProduct", message: "Use storeProduct instead")
+    @available(macOS, obsoleted: 1, renamed: "storeProduct", message: "Use storeProduct instead")
```

Comment:
I would remove the `message`, the fix it will provide basically the same thing thanks to the `renamed` parameter.

Reply:
I wasn't sure what to do about this one, tbh, since it's not a _rename_, it's an entirely different type

Reply:
Oooh that makes sense, then I would leave it. Maybe capital S?

> Use `StoreProduct` instead?

Reply:
good call!